### ISSUE TITLE
latexmk Fatal error: Invalid Version: 4.43a.0

### ISSUE THREAD

**Issue by LUXUV587** [sharelatex/sharelatex#302]
This error seems to be an old issue when I ran `grunt run:all`.

```
[2015-04-23T01:12:53.117Z]  WARN: clsi/35842 on luxulatex: timeouts and sandboxing are not 
enabled with CommandRunner

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:1011:11)
    at Process.ChildProcess._handle.onexit (child_process.js:802:34)
Warning: -> error 8 /var/www/sharelatex/clsi/app.js (60206ms) Use --force to continue.

Aborted due to warnings.
```
So, I ran `grunt check --force` and following error appears:
```
Running "check:redis" task
Checking Redis is running... OK.
Running Redis version 3.0.0

Running "check:latexmk" task
Checking latexmk is installed... Fatal error: Invalid Version: 4.43a.0
```
Thus, I stuck at 'Sorry, something went wrong and your project could not be compiled.', but everything else ok.

How to figure out this error?

**Comment by hulk1st** [comment]
Hi, I have the same version of latexmk and also get the invalid version after grunt check. Still it works. Must be something else in your case.

**Comment by LUXUV587** [comment]
Yes, I installed again and this time it works well.

**Comment by egeu5** [comment]
I've got the same issue. Clean reinstall is not working at the moment.

**Comment by LUXUV587** [comment]
I am a newbie. In my own case, I found that the redis would not run automatically. You can refer to http://redis.io/topics/quickstart ,https://gist.github.com/bdotdub/714533 and http://bespokebytes.com/start-getting-up-and-running-with-upstart/ to run redis.

**Comment by egeu5** [comment]
Redis is running. I tried it with executing the redis-cli and typing ping and get a PONG. Also I tried to set a value and get it back. Everything with redis seems to work fine.

**Comment by jpallen** [comment]
I think that the 'a' in the version number is confusing our check, but if you have latexmk installed it should still work fine.

**Comment by muelli** [comment]
I having the same issue. I have manually modified my Ubuntu's latexmk 4.43a to remove the "a" suffix. Now it works better.

I'd appreciate the fix in #318.
